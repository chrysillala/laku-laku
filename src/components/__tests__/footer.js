import React from 'react'
import renderer from 'react-test-renderer'

import Footer from '../footer'

describe("Footer", () => {
  it("footer renders correctly", () => {
    const tree = renderer
      .create(<Footer footerText="Footer Text" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})