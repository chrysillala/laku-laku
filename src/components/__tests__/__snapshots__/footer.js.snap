// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer footer renders correctly 1`] = `
<footer
  css={
    Object {
      "map": undefined,
      "name": "r1nipy",
      "next": undefined,
      "styles": "
      background-color: turquoise;
      color: white;
      padding: 2rem;
      text-align: center
      ",
      "toString": [Function],
    }
  }
>
  Â©
   
  2021
   
  Footer Text
</footer>
`;
